framework/wazuh/rbac/orm.py

```diff
@@ -1348,11 +1384,11 @@ def add_policy(self, name: str, policy: dict, check_default: bool = True):
         try:
             if policy is not None and not json_validator(policy):
                 return SecurityError.ALREADY_EXIST
```

Comment:
This is legacy code. I already reported that there were some parts that did not make much sense, but analyzing this whole logic, this use case would never occur since we have proper checks on higher scopes (the API models for instance). I didn't believe fixing this kind of stuff was part of this issue scope.

Reply:
I understand, but the fix should just be to return INVALID instead of ALREADY_EXIST?

Reply:
To be done in new opened issue